manifests/03-clusterrole.yaml

```diff
@@ -353,6 +353,12 @@ rules:
     verbs:
       - get
       - list
+  - apiGroups:
```

Comment:
This can be removed as the communication is done with HTTP and no longer with a `pod/exec`.